### PR TITLE
Fixes #22385 - Add cp_label to API

### PR DESCRIPTION
**Pull request by xprazak2**

### CODE REVIEW

File: app/views/katello/api/v2/repositories/show.json.rabl [hunk 1]
```diff
@@ -22,6 +22,7 @@ attributes :upstream_username
 attributes :ostree_upstream_sync_policy, :ostree_upstream_sync_depth, :compute_ostree_upstream_sync_depth => :computed_ostree_upstream_sync_depth
 attributes :deb_releases, :deb_components, :deb_architectures
 attributes :ignore_global_proxy
+attributes :cp_label
```
Comment (jlsherrill) [inline app/views/katello/api/v2/repositories/show.json.rabl]: could we rename this to 'content_label' or something a little more descriptive?

**Comment by xprazak2** [comment]
I renamed it to `:content_label`

**Comment by jlsherrill** [comment]
chatting with the user that requested this, it seems that they actually wanted it in the index api, mind moving it to base.json.rabl?

**Comment by xprazak2** [comment]
Done, I moved it into `base.json.rabl`

**Pull request merged**